src/applications/gi/sass/partials/_gi-vet-tec.scss

```diff
@@ -22,6 +22,12 @@
 
   $single-column-width: 641px;
 
+  @media only screen and (max-width: 767px) {
+    .vettecLogoDesktop {
```

Comment:
Please change the class name to use dashes instead of camelCase.

Reply:
Updated class name to vettec-logo-desktop